src/main/scala/com/github/unchama/seichiassist/subsystems/dragonnighttime/System.scala

```diff
@@ -0,0 +1,18 @@
+package com.github.unchama.seichiassist.subsystems.dragonnighttime
+
+import cats.effect.IO
+import com.github.unchama.seichiassist.concurrent.PluginExecutionContexts
+import com.github.unchama.seichiassist.meta.subsystem.StatefulSubsystem
+import com.github.unchama.seichiassist.subsystems.dragonnighttime.bukkit.task.global.DragonNightTimeRoutine
+
+object System {
+  def wired[F[_]]: StatefulSubsystem[F, List[IO[Nothing]]] = {
```

Comment:
`IO` に依存せずとも、
```suggestion
  def wired[F[_], G[_]: Sync: Timer]: StatefulSubsystem[F, List[G[Nothing]]] = {
```
として `DragonNightTimeRoutine.apply` を多相的にすれば良さそう